docs/porting/third-party-deps.md

```diff
@@ -30,24 +30,29 @@ netstandard1.4
 netstandard1.5
 netstandard1.6
 netcoreapp1.0
-portable-<tfm_list>-whatever
+portable-net45-win8
+portable-win8-wpa8
+portable-net451-win81
+portable-net45-win8-wpa8-wpa81
 ```
 
-These are the Target Framework Monikers (TFM for short) which map to versions of [The .NET Standard Library](../standard/library.md).  Note that `netcoreapp1.0`, while compatible, is for applications and not libraries.  Although there's nothing wrong with using a library which is `netcoreapp1.0`-based, that library may not be intended for anything *other* than consumption by other `netcoreapp1.0` applications.
+These are the Target Framework Monikers (TFM for short) which map to versions of [The .NET Standard Library](../standard/library.md) and traditional Portable Class Library (PCL) profiles which are compatible with .NET Core.  Note that `netcoreapp1.0`, while compatible, is for applications and not libraries.  Although there's nothing wrong with using a library which is `netcoreapp1.0`-based, that library may not be intended for anything *other* than consumption by other `netcoreapp1.0` applications.
```

Comment:
nit 2: the article The in the standard Library link should be lower caps and ideally out of the link